pkg/command/context.go

```diff
@@ -1195,10 +1199,10 @@ func useCtx(cmd *cobra.Command, args []string) error {
 		}
 	}
 	if suffixString != "" {
-		suffixString = " (" + suffixString + ")"
+		suffixString = "(" + suffixString + ")"
 	}
 
-	log.Infof("Successfully activated context '%s' of type '%s'%s.", ctxName, ctx.ContextType, suffixString)
+	log.Infof("Activated context '%s' %s successfully ", ctxName, suffixString)
```

Comment:
Now that `successfully` is at the end, it gets lost in all the other printouts.
But it is not very important anyway so I suggest we remove it and simply print:
`Activated context 'testprojectsyncer' (Type: tanzu, Project: newProjectTestne, ProjectID: 3351bd40-5150-46f7-96c5-d368b5f99932, Space: store-dev)`